src/main/webapp/app/components/tabs/CurationHistoryTab.tsx

```diff
@@ -21,29 +20,9 @@ export interface ICurationHistoryTabProps extends StoreProps {
   historyData: FlattenedHistory[];
 }
 
-type HistoryTabData = {
-  record: HistoryRecord;
-  timeStamp: number;
-  admin: string;
-  location: string;
-  objectField?: string;
-};
-
 const CurationHistoryTab = observer(({ historyData, getUsers, users, historyTabStore, getDrugs }: ICurationHistoryTabProps) => {
   const [drugList, setDrugList] = useState<IDrug[]>([]);
 
-  function findObjectFieldsInRecord(record: HistoryRecord) {
```

Comment:
@bprize15 I saw that this wasn't used anymore, so I removed. Let me know if I shouldn't.

Reply:
Yeah, thanks for removing that. I must've forgot to take it out during the history refactor